src/main.py

```diff
@@ -0,0 +1,54 @@
+import argparse
+
+from pipeline.eval import eval_on_records
+from utils import get_steps_right
+
+def main(args):
+    if args.load_in_8bit is True \
+        and args.load_in_4bit is True:
+        raise ValueError("both load_in_8bit and load_in_4bit are set. "
+                         "only one of them should be set at a time")
+    
+    valid, input_steps = get_steps_right(args.steps)
+    if valid:
+        if "ft" in input_steps:
+            pass
```

Comment:
These can be made more readable.

Reply:
I left it `pass` for the future implementation. Could you elaborate?

Reply:
"ft" is a little too short of a name. Spelling it out fully will be more useful ("fine-tuning").